docs/events/api/timeline-api-overview.asciidoc

```diff
@@ -4,3 +4,5 @@
 
 You can create timelines and timeline templates via the API, as well import new
```

Comment:
Suggesting some minor tweaking:
You can create timelines and timeline templates via the API, and import new